src/plugins/dashboard/public/dashboard_container/embeddable/dashboard_container.tsx

```diff
@@ -876,7 +876,7 @@ export class DashboardContainer
     this.dispatch.setScrollToPanelId(id);
   };
 
-  public scrollToPanel = async (panelRef: HTMLDivElement) => {
+  public scrollToPanel = async (panelRef: HTMLDivElement, editMode?: boolean) => {
```

Comment:
Why have an "edit mode" prop when the only place `scrollToPanel` is used is when editing a panel?